knapsack-cloud/sandbox-starter
# Fix: style dictionary build

This PR fixes an issue where the rollup config was creating a circular compilation chain. We are now using nodemon to watch our design tokens for changes and then using the some new `sd` scripts.

**To test:**
1. Pull this PR down
2. Run `yarn install`
3. run `yarn build`
4. Verify —> Everything compiles as expected
